unit_tests/src/unit_test_utils.hpp

```diff
@@ -78,7 +78,9 @@ namespace unit_test_utils
         ufs_info.host_name = host_name;
         ufs_info.vault_path = vault_path;
 
-        return adm::client::add_resource(_comm, ufs_info).value() == 0;
+        adm::client::add_resource(_comm, ufs_info);
+
+        return true; // TODO
```

Comment:
Is this TODO referring to the fact that this function now throws? Or was that always the case?

Reply:
I think that was removed in the second commit.

Reply:
Yep... I forgot to hit the next button. :)